CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new image viewer that appears when you tap an image.
 - Included the npub in the properties list sent to analytics.
 - Removed the like and repost counts from the Main and Profile feeds.
-- Replaced hard-coded color values.
```

Comment:
Sorry for the confusion earlier -- this line was duplicated below in the 0.1.25 section. It should not be there (it wasn't released in 0.1.25), but it should be here since it hasn't been released yet. So we'll want to undo this delete.

Reply:
Copy that. I'll put it back!